test/kubemark/start-kubemark.sh

```diff
@@ -39,6 +39,47 @@ KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
 KUBEMARK_DIRECTORY="${KUBE_ROOT}/test/kubemark"
 RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 LOCAL_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark"
+LOCAL_KUBECONFIG_TMP="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tmp"
+
+export SCALEOUT_TP_COUNT="${SCALEOUT_TP_COUNT:-1}"
+
+### files to explicitly save the kubeconfig to different cluster or proxy
+### those are used for targeted operations such as tenant creation etc on
+### desired cluster directly
+###
+TP_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark-tp"
+PROXY_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.proxy.saved"
+RP_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.rp.saved"
+if [[ "${SCALEOUT_CLUSTER:-false}" == "false" ]]; then
+  TP_KUBECONFIG="${LOCAL_KUBECONFIG}"
+  LOCAL_KUBECONFIG_TMP="${LOCAL_KUBECONFIG}"
+  RP_KUBECONFIG_SAVED="${LOCAL_KUBECONFIG}"
+fi
+
+
+### POC tenant yaml files
+###
+TENANT1_YAML="${KUBE_ROOT}/perf-tests/clusterloader2/testing/arktos/tenant1.yaml"
```

Comment:
As the number of TPs could be a number other than 2, it does not make much sense to create two test tenants. 

I suggest to remove anything about test tenant and let the tester create the test tenants on the fly.

Reply:
This PR is to port what's currently in POC branch to deploy scaleout back up to master. We can make fixes and big improvements in separate PR either after we merge everything to master and work there, of if we have a tracking work-item for this near completion.